content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -0,0 +1,93 @@
+=====================
+Inventory Adjustments
+=====================
+
+Inventory adjustments can be accessed through :menuselection:`Inventory --> Operations -->
+Inventory Adjustments`.
+
+The report has all products currently in the stock. Each line is a unique combination of
+
+- Location
+- Product
+- Lot/Serial
+- Package
+- Owner
+
+Add new product in your stock
+=============================
+
+To add a line, just hit *Create* and fill in product and counted quantity fields then press `tab`
+or click *Save*
+
+.. image:: count_products/create-new-inventory-adjustment-line.png
+   :align: center
+   :alt: Create a new inventory adjustment line for a product and fill in the fields as desired
+
+At this stage, the count is recorded but not yet *applied*. Meaning the quantity on hand is not yet
+updated.
+
+Just press *Apply* on the line or button *Apply All* button on the top of the screen.
+
+*Apply All* allows to edit the reason to use for the adjustments applied
+
+.. image:: count_products/apply-inventory-adjustment.png
+   :align: center
+   :alt: Apply all wizard applies the inventory adjustment once a reason is specified.
+
+Count products
+==============
+
+Counting products is a recurring activity in a warehouse. It often happens that theoric stock
+levels are not correct and must be amended.
+
+If a count is correct, press on *set*, it will copy the on hand quantity in the counted quantity.
+Applying such line will record a move in the history with a 0 quantity.
+
+If the count is different, record it in the counted quanditity field. Applying such line will
+record a move with the difference in tin the history.
+
+.. image:: count_products/history-inventory-adjustments.png
+   :align: center
+   :alt: Inventory Adjustments History dashboard detailing a list of prior product moves.
+
+.. note::
+   It happens that users count but apply only later. In the meantime, transfers of a counted product
+   can occur. In such case quanities counted and on hand are not consistant with the difference. When
+   applying such line, Odoo will ask if you want to keep the difference and the count.
+
+Plan counts
+===========
+
+On each line there is:
+
+- *Scheduled Date*: it's the date at which a count should be made.
+- *User*: it's the person in charge of the count.
+- *Accounting Date*: this is the date at which the adjustments will be accounted. The column is
+  hidden by default.
+
+.. image:: count_products/planned-inventory-adjustments.png
+   :align: center
+   :alt: A schedule of upcoming Inventory Adjustments and when they'll be applied in accounting.
+
+.. important::
+   In the barcode app, you will only see counts to make assigned to you, scheduled for today or
+   earlier.
+
+If you want to plan big counts, the easiest way is to select some lines, then hit **Request A
```

Comment:
- Delete the three sentences on lines 76-78 `If you want to...with the on hand quantity.`
- Replace with: 

> `To plan big counts, select the desired product lines on the :guilabel:'Inventory Adjustments' page. Then, click :guilabel:'Request a Count'. Fill in the following information:`
> `- :guilabel:'Inventory Date': the planned date of the count.`
> `- :guilabel:'User': the user responsible for the count.`
> `- :guilabel:'Accounting Date': the date at which the inventory adjustment will be accounted.`
> `- :guilabel:'Count': to leave the :guilabel:'On Hand Quantity' of each product line blank, select :guilabel:'Leave Empty'. To prefill the :guilabel:'On Hand Quantity' of each product line with the current value recorded in the database, select :guilabel:'Set Current Value'.`
> `Finally, click :guilabel:'Confirm' to request the count.`

Reply:
Made small change

Instead of: 
```
Then, click :guilabel:`Request a Count`. Fill in the following information:
``` 

I did: 
```
Then, click :guilabel:`Request a Count` and fill in the following information:
```